source/infrastructure/disaster-recovery/index.html.md.erb

```diff
@@ -75,6 +85,7 @@ We often rely on Azure for:
 - Terraform state in Azure Storage
 - Infrastructure and application secrets in Azure Key Vault
 - Daily production database backups in Azure Storage
+- some applications use Azure Storage for storing or uploading data
```

Comment:
This impacts the running application, not the delivery. It should be covered in the other sections

Reply:
moved